src/Models/Environment.php

```diff
@@ -168,10 +168,15 @@ public function cloneDatabase(Environment $from_env, array $options = [])
         if (isset($options['updatedb'])) {
             $options['updatedb'] = (int)$options['updatedb'];
         }
-        $params = array_merge(
-            ['from_environment' => $from_env->getName(),],
-            $options
-        );
+        $params = [
+            'from_environment' => $from_env->getName(),
+            'updatedb' => $options['updatedb'] ?? 0,
+            'clear_cache' => $options['clear_cache'] ?? false,
+        ];
+        if (!empty($options['from_url']) && !empty($options['to_url'])) {
+            $params['wp_replace_siteurl']['from_url'] = $options['from_url'];
+            $params['wp_replace_siteurl']['to_url'] = $options['to_url'];
+        }
```

Comment:
So, was it something about WP search-replace that was causing the long tests to fail?

Reply:
That was one of the failing tests, probably broken since it was created and never noticed.

There were other failing tests also fixed in this PR